### ISSUE TITLE
Uncalled-for indentation of block after <<

### ISSUE THREAD

**Issue by forthrin** [rubocop/rubocop#15432]
### Before submitting

- [x] I have searched the existing issues and this hasn't been reported yet.
- [x] I am running the latest stable version of RuboCop.

### Issue type

False positive

### Cop name

Layout/ElseAlignment

### Steps to reproduce the problem

```ruby
foo << bar.each do
  1
rescue StandardError
  2
else
  3
end

```

### Expected behavior

Shouldn't this be accepted as-is?

### Actual behavior

Align `else` with `bar.each`

### RuboCop configuration

```yaml

```

### RuboCop version

```shell
System Version: macOS 26.5.1 (25F80)
Model Identifier: MacBookAir10,1
ruby 4.0.5 (2026-05-20 revision 64336ffd0e) +PRISM [arm64-darwin25]
rubocop     2026-07-05 c1451b3d9
```
